gildniy/WayFarer
# Fix UI (Challenge 1)

#### What does this PR do?
- Remove test as a user  and test as an admin and link it to the login and register pages
- Remove all the extra whitespace
- All buttons have the consistency of color
- Remember me is distortion is fixed
- Change enter to login
- Instead of the word delete, used the delete icon
#### Description of Task to be completed?
Fix the UI For Challenge 1
#### How should this be manually tested?
As a user visiting the [gh-page](https://gildniy.github.io/WayFarer/UI/):
- As a user by getting to the UI [gh-page](https://gildniy.github.io/WayFarer/UI/index.html), when you click on the Login button on the login you should be redirected to the dashboard as a Regular User, and when on the Register form, when clicking on Register button you should be redirected to the dashboard as an admin user
- The white spaces in the dashboard between navigation, main content and footer have been changed
- The main buttons are in green, the ones which are not too important are in a grey color, and the delete button is in red color (On single trip page for the Admin user)
- The remember me checkbox is now in a 1 element not duplicated as before
- The Login Button Text has been changed from `ENTER` to `LOGIN`
- As a regular user, the deleted text on the bookings list is changed into a recycle bin icon
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#167752982, #167752996, #167753002, #167753019, #167753027, #167753051
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
